client/my-sites/plans-features-main/index.tsx

```diff
@@ -329,16 +329,9 @@ const PlansFeaturesMain = ( {
 	const showEscapeHatch =
 		intentFromSiteMeta.intent && ! isInSignup && defaultWpcomPlansIntent !== intent;
 
-	const {
-		isLoading: isLoadingSimplifiedFeaturesGridExperiment,
-		isTargetedView: isTargetedViewForSimplifiedFeaturesGridExperiment,
-		variant: simplifiedFeaturesGridExperimentVariant,
-		setVariantOverride: setSimplifiedFeaturesGridExperimentVariantOverride,
-	} = useSimplifiedFeaturesGridExperiment( {
-		flowName,
-		isInSignup,
-		intent,
-	} );
+	const isTargetedSignupFlow = isInSignup && flowName === 'onboarding';
+	const isTargetedAdminIntent = ! isInSignup && intent === 'plans-default-wpcom';
+	const showSimplifiedFeatures = isTargetedSignupFlow || isTargetedAdminIntent;
```

Comment:
This is temporary and should be removed. The "simplified" plans grid is the default and should be shown in all places. This can be removed once we have clarity on p5uIfZ-ggB-p2#comment-24241